docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. JavaScript provides benefits like :- 
+* Less server interaction as in validating user input before sending page to server, thus, reducing server traffic.
+* Immediate feedback to the site visitors so that they dont reload page just to find they forgot to enter something.
+* Increased interactivity and Richer interfaces facilitating users to experience mouse hover changes, drag-drop components, animations etc.
+
+Client-Side JavaScript which enables the enhancement and manipulation of web pages and client browsers, is commonly used. The script can be referenced by or included in the HTML and code is interpreted by the browser. This takes the plain static HTML page to a level where user can interact to it.
+
+Server-Side JavaScript(Node.js) is another version of JavaScript that enables back-end access to databases, file systems, and servers. It is built on Google Chrome's JavaScript V8 Engine. It uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node can be a great solution for applications requiring I/O bound operations, data streaming etc. More details can be found  [here](https://nodejs.org/en/about/)
```

Comment:
Server side JS isn't really "another version" of JavaScript.  It's the same language running in a different environment.  
Node is built on V8, but  JavaScript is not.  This paragraph starts out talking about Javascript but then every later reference to 'it' refers to Node.  Maybe change 'It' in the second and third sentences to 'NodeJS'.